### PR TITLE
New virology chems. Makes virus food actually useful. Adds uranium to virology uses.

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: 
rscadd: Adds Uranium to Virus reaction. Will create a random symptom between 5 and 6 when mixed with blood.
rscadd: Adds Virus rations, which creates a level 1 symptom when mixed with blood.
rscadd: Adds Mutagenic agar, which creates a level 3 symptom.
rscadd: Adds Sucrose agar which creates a level 4 symptom.
rscadd: Adds Weak virus plasma, which creates a level 5 symptom.
rscadd: Adds Virus plasma, which creates a level 6 symptom.
rscadd: Adds Virus food and Mutagen reaction, creating Mutagenic agar.
rscadd: Adds Virus food and Synaptizine reaction, creating Virus rations.
rscadd: Adds Virus food and Plasma reaction, creating Virus plasma.
rscadd: Adds Synaptizine and Virus plasma reaction, creating weakened virus plasma.
rscadd: Adds Sugar and Mutagenic agar reaction, creating sucrose agar.
rscadd: Adds Saline glucose solution and Mutagenic agar reaction, creating sucrose agar.
rscadd: Adds Viral self-adaptation symptom, which boosts stealth and resistance, but lowers stage speed.
rscadd: Adds Viral evolutionary acceleration, which bosts stage speed and transmittability, but lowers stealth and resistance.
/:cl:

These changes make virology less RNG intensive.
Virus food + mutagen = Mutagenic agar.
Virus food + synaptizine = virus rations.
Virus food + plasma = virus plasma.
Synaptizine + Virus plasma = weakened virus plasma
Sugar OR saline glucose solution + mutagenic agar = sucrose agar

Uranium = level 5 or 6 symptom
Mutagenic agar = level 3 symptom
Sucrose agar = level 4 symptom
Weak virus plasma = level 5 symptom
Plasma virus = level 6 symptom
Virus rations = level 1 symptom

Old stuff, ignore below:
_Toxin now creates a symptom between 3-5 and Plasma now creates a symptom between 5-6.

This should make virology less RNG by allowing you to get the more exclusive levels 5 and 6 viruses easier using plasma._

### CODE REVIEW

**Comment by GunHog** [comment]
Thank you :+1: (Please make sure you tested this. Hard.)

**Comment by LanCartwright** [comment]
Cough - 1
Headache - 1
Itch - 1
Sneeze - 1
Fever - 2
Shivering - 2
Choking - 3
Antibodies - 3
Longevity - 3
Stims - 3
Vomit - 3
Weakness - 3
Weight loss - 3
Blood vomit - 4
Beard - 4
Confusion - 4
Weight gain - 4
Weight even - 4
Tox Comp - 4
Deafness - 4
Dizziness - 4
Alopecia - 4
Vitiligo - 4
Revitiligo - 4
Ocular restore 4
Hallucigen  - 5
DAR - 5
Hyphema - 5
Eternal youth - 5
Combustion - 6
Flesheating - 6
DAS - 6
ToxFilter - 6
SelfResp - 6
Voice change - 6

**Comment by LanCartwright** [comment]
Will post before and after odds tomorrow after calculation.

**Comment by Buggy123** [comment]
:+1:

**Comment by PKPenguin321** [comment]
only problem i see with this is that afaik "toxin" is for the most part a non-renewable resource, but other than that it's good

**Comment by LanCartwright** [comment]
Before:
Virus food: 6 viruses
Mutagen: 21 viruses
Plasma: 22 viruses

After:
Virus food: 6 viruses.
Mutagen: 21 viruses
Toxin: 23 viruses
Plasma: 10 viruses.

What I'm thinking now, is perhaps another reagent to cover symptoms 3 and 4 exclusively, and an additional reagent that isn't avaliable roundstart to cover ONLY symptoms 6.

**Comment by lordpidey** [comment]
>and an additional reagent that isn't avaliable roundstart to cover ONLY symptoms 6.

How about uranium?

**Comment by Xhuis** [comment]
How does this make any sense? Toxins are meant to poison blood, and would make no sense to induce random mutations. The only reagents that create new symptoms are logical: virus food is meant to cultivate the virus, mutagen is meant to mutate blood, and plasma reacts with micro-organisms in blood to create symptoms.

**Comment by LanCartwright** [comment]
it doesn't. at this point it's a placeholder. I might have to add a new reagent alltogether.

@lordpidey I think I'll add that to "level 6 symptoms only". But roundstart uranium apparently leads to roundstart EMPs.

**Comment by LanCartwright** [comment]
Here's what I'm going to do instead:
Buff/change to virus food:
Adding mutagen to virus food will create Virus mutagenic agar which creates a virus of level 3 flat.
Adding plasma to virus food will create Virus plasma which creates a virus of 6 flat.
Adding Synaptizine to Virus plasma creates weakened virus plasma, which creates a virus of 5 flat.
Adding PLACEHOLDER (considering saline glucose) to virus mutagenic agar creates sucrose agar, which creates a virus of level 4 flat.

It shouldn't be too complex, but it'll make virus food more useful and virology more interesting and less RNG intense in the right hands.

**Comment by Xhuis** [comment]
By "flat" do you mean that it would always create a symptom with the range described? i.e. Virus plasma always makes a level 6 symptom? The reagent names are iffy and the virus plasma might be too easy to make, but other than that, it seems like a really good change for making virology less boring. :+1:

**Comment by LanCartwright** [comment]
Yes.
6 flat means level 6 viruses only, and so on so forth.

**Comment by GunHog** [comment]
This is good. It will be fun to toy around with my reagents like a chemist, and this serves as a very good counter to the increase of possible unwanted symptoms added recently. I feel I can have a lot more control over where I want to go with my virus rather than it being almost entirely RNG.

**Comment by LanCartwright** [comment]
No? Maybe?

You are able to use a beaker to get some of the 1000u of virus food out though.

**Comment by lordpidey** [comment]
> Adding PLACEHOLDER (considering saline glucose) to virus mutagenic agar creates sucrose agar, which creates a virus of level 4 flat

How about just sugar?

Making it just sugar will allow for botany/chef to do some hilarious ghetto virology, if they can get infected blood and get mutagen from the chemist.

**Comment by LanCartwright** [comment]
problem is, chemists in general are the absolute worst, and I'd rather not they be relied upon.

**Comment by Xhuis** [comment]
Candy has sugar, and the virologist has a blender if I'm not mistaken.

**Comment by LanCartwright** [comment]
he does. I'll do both.

**Comment by newfren** [comment]
Pretty sure chemistry has sugar.

**Comment by LanCartwright** [comment]
@newfren 
> problem is, chemists in general are the absolute worst, and I'd rather not they be relied upon.

Given how many departments rely on chemistry, I'd personally not like the player to be forced into running to another subdepartment for one reagent.

**Comment by newfren** [comment]
I've never seen a chemist refuse to give me something as a virologist. Might be the psychology of us both being medbay people because it happens loads as bartender/botanist.

**Comment by Xhuis** [comment]
Most experienced chemists have an emotional spectrum ranging from grumpy to frustrated. You have to realize that the typical chemist gets _so_ much shit. Anyone gets poisoned and the chemists are either strip-searched or even lynched based off of little to no evidence. People wordlessly break into your workplace if you leave for more than three seconds, then scream for Security when you syringe them with a stun mix to get them out. Assistants and people that are bored make stupid, pointless requests.

I mainly played chemist back in the day (with a bit of Virology for good measure), and I wouldn't hesitate to reject half the requests that came my way without proper reasoning. I never typically made chemicals for the virologist other than once - I just told him to get Chemistry access and that he has my approval, because I'm typically busy pumping out medicine for the issues on the station or trying to convince the CMO that salicyclic acid is not poison and that I'm not trying to murder everyone by giving it to them.

**Comment by Iamgoofball** [comment]
One of the reasons I added the poison kit was so that people would stop assumimg toxloss = lynch chemist

**Comment by LanCartwright** [comment]
@KorPhaeron why does this have a DNM

**Comment by KorPhaeron** [comment]
I forget. I think I accidentally labelled this one when I was telling you to test the deadly symptoms

**Comment by GunHog** [comment]
If Kor can remember to test, than so can you! :D

**Comment by LanCartwright** [comment]
I'll look into this, then asthmothsia. This really shouldn't need that much testing because it essentially makes a few reagents for specialised levels in viruses.

Anyway, after this and asthmothsia, I'll work on an actual traitor virus.
